tools/transferoffsets/main.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	"github.com/Shopify/sarama"
+	"github.com/wvanbergen/kazoo-go"
+)
+
+var (
+	zookeeper = flag.String("zookeeper", os.Getenv("ZOOKEEPER_PEERS"), "The zookeeper connection string")
+	groupName = flag.String("group", "", "The consumer group to transfer offsets for")
+)
+
+func main() {
+	flag.Parse()
+
+	if *zookeeper == "" {
+		log.Fatal("The -zookeeper command line argument is required")
+	}
+
+	if *groupName == "" {
+		log.Fatal("The -group command line argument is required")
+	}
+
+	var (
+		config         = kazoo.NewConfig()
+		zookeeperNodes []string
+	)
+
+	zookeeperNodes, config.Chroot = kazoo.ParseConnectionString(*zookeeper)
+	kz, err := kazoo.NewKazoo(zookeeperNodes, config)
+	if err != nil {
+		log.Fatal("[ERROR] Failed to connect to the zookeeper cluster:", err)
+	}
+	defer kz.Close()
+
+	brokerList, err := kz.BrokerList()
+	if err != nil {
+		log.Fatal("[ERROR] Failed to retrieve Kafka broker list from zookeeper:", err)
+	}
+
+	group := kz.Consumergroup(*groupName)
+	if exists, err := group.Exists(); err != nil {
+		log.Fatal(err)
+	} else if !exists {
+		log.Fatalf("[ERROR] The consumergroup %s is not registered in Zookeeper", *groupName)
+	}
+
+	if instances, err := group.Instances(); err != nil {
+		log.Fatal("[ERROR] Failed to get running instances from Zookeeper:", err)
+	} else if len(instances) > 0 {
+		log.Printf("[WARNING] This consumergroup has %d running instances. You should probably stop them before transferring offsets.", len(instances))
```

Comment:
so we log this and then just go ahead a transfer immediately without letting the user actually say "oh, maybe I will stop them first"?